src/Microsoft.IdentityModel.Tokens/TokenUtilities.cs

```diff
@@ -243,6 +243,16 @@ internal static IEnumerable<Claim> MergeClaims(IEnumerable<Claim> claims, IEnume
             return result;
         }
 
+        /// <summary>
+        /// Check whether the given exception type is recoverable by LKG.
+        /// </summary>
+        /// <param name="exception">The exception to check.</param>
+        /// <returns><c>true</c> if the exception is certain types of exceptions otherwise, <c>false</c>.</returns>
+        internal static bool IsRecoverableException(Exception exception)
```

Comment:
Should this be in internal api doc?

This should have a comment about its purpose: not used, here for compat.

Reply:
It's already in the internal shipped API txt file. Never got removed from there.
Updated the comment.